02 Algorithm Reference/01 Overview/01 Introduction.html

```diff
@@ -21,13 +21,13 @@
 		</span>
 </p>
 <p>
-	<code class="csharp">Portfolio</code> <code class="python">self.Portfolio</code> is a dictionary of SecurityHolding classes. 
-	These classes track the individual portfolio items profit and losses, fees and quantity held. e.g. 
+	$[Portfolio,P:QuantConnect.Algorithm.QCAlgorithm.Portfolio] is a dictionary of SecurityHolding classes.
+	These classes track the individual portfolio items profit and losses, fees and quantity held. e.g.
 	<code class="csharp fsharp">Portfolio["IBM"].LastTradeProfit</code>
 	<code class="python">self.Portfolio["IBM"].LastTradeProfit</code>.
 </p>
 <p>
-	Other helpers like <code>Transactions</code>, <code>Schedule</code>, <code>Notify</code>, and <code>Universe</code> have their
+	Other helpers like $[Transactions,P:QuantConnect.Algorithm.QCAlgorithm.Transactions], $[Schedule,P:QuantConnect.Algorithm.QCAlgorithm.Schedule], $[Notify,P:QuantConnect.Algorithm.QCAlgorithm.Notify], and $[Universe,T:QuantConnect.Data.UniverseSelection.Universe] have their
```

Comment:
In this case, `Universe` is `P:QuantConnect.Algorithm.QCAlgorithm.Universe` instead of `T:QuantConnect.Data.UniverseSelection.Universe`, since we are talking about the helpers that are properties (P).